_pytest/python.py

```diff
@@ -1104,8 +1104,7 @@ def raises(expected_exception, *args, **kwargs):
     ``message`` to specify a custom failure message::
 
         >>> with raises(ZeroDivisionError, message="Expecting ZeroDivisionError"):
-        ...    pass
-        ... Failed: Expecting ZeroDivisionError
+        ...    1/0
```

Comment:
Oops wait, your are correct, I must have that confused with another example. Will fix it.